### PR TITLE
ROC - 4762 Remove pagination for party type individual and company

### PR DESCRIPTION
**Pull request by rattur**
### JIRA link

https://tools.hmcts.net/jira/browse/ROC-4762

### Change description

Remove pagination (extra page) after viewing defendant response for party type individual and company only for part admission

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ x] No

### CODE REVIEW

File: src/main/features/claimant-response/views/defendants-response.njk [hunk 1]
```diff
@@ -1,10 +1,6 @@
 {% extends "layout.njk" %}
 
-{% if claim.response.responseType === ResponseType.PART_ADMISSION.value and page === 1 %}
-  {% set heading = 'How they want to pay' %}
-{% else %}
-  {% set heading = 'The defendant’s response' %}
-{% endif %}
+{% set heading = 'The defendant’s response' %}
```
Comment (varma02) [inline src/main/features/claimant-response/views/defendants-response.njk]: the code you removed, is that not necessary? we don't need different heading for PART_ADMISSION? 

i didn't find anything in the story to make this change...could you please explain??

File: src/main/features/claimant-response/views/defendants-response.njk [hunk 2]
```diff
@@ -77,81 +73,64 @@
...
-              paymentIntentionFragment(
-                amount = claim.response.amount,
-                paymentIntention = claim.response.paymentIntention
-              ) }}
-            </p>
+            paymentIntentionFragment(
+            amount = claim.response.amount,
+            paymentIntention = claim.response.paymentIntention
+            ) }}
+          </p>
```
Comment (varma02) [inline src/main/features/claimant-response/views/defendants-response.njk]: Did you remove tab spacing? why?

File: src/main/features/claimant-response/views/defendants-response.njk (hunk 1 above)
Comment (rattur) [reply src/main/features/claimant-response/views/defendants-response.njk]: Ok well the story didn't explain things and was incorrect so I have left notes on the story but in general the change is to remove the additional pagination (the extra page) in the view defendant response for party type individual and company only for part admission and redirect to task list page

File: src/main/features/claimant-response/views/defendants-response.njk (hunk 2 above)
Comment (rattur) [reply src/main/features/claimant-response/views/defendants-response.njk]: Not sure what you mean?

**Pull request closed without merging**